src/smvlang/expr2smv.h

```diff
@@ -8,6 +8,5 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <util/expr.h>
 
-bool expr2smv(const exprt &expr, std::string &code);
-bool type2smv(const typet &type, std::string &code);
-
+bool expr2smv(const exprt &, std::string &code, const namespacet &);
+bool type2smv(const typet &, std::string &code, const namespacet &);
```

Comment:
Not for this PR, but rather asking about potential future work: Is `bool` really the appropriate return type? Shouldn't places that `return true` really be using `UNIMPLEMENTED` instead? Then the return type could instead be `std::string`.

Reply:
Another option would be to send anything that that code doesn't know how to format to `format_expr`.